File: MPI/mpicheck/tests/functions/1.0/MPI_Type_contiguous.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
       INTEGER var_3
        call mpi_type_contiguous(var_0, var_1, var_2, var_3)
       call pmpi_type_contiguous(var_0, var_1, var_2, var_3)
        end program main
    

File: MPI/mpicheck/tests/functions/1.0/MPI_Type_contiguous.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int var_0;
    MPI_Datatype var_1;
    MPI_Datatype *var_2;
    int ret;
    /* calls */
    ret = MPI_Type_contiguous(var_0, var_1, var_2);
    ret = PMPI_Type_contiguous(var_0, var_1, var_2);
    return 0;
}
